man/systemd.service.xml

```diff
@@ -729,6 +729,18 @@
         <xi:include href="version-info.xml" xpointer="v246"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TimeoutMountSec=</varname></term>
```

Comment:
Hmm, do we really need this? I think it's OK to use the reload timeout here, especially given that we'd probably refresh more stuff in main process's namespace during reload in the future.

Reply:
@poettering said to do the same as SERVICE_CLEANING for this, if he's fine with removing this I'll remove it

Reply:
i am fine either way. i kinda sense that sooner or later people want a separate timeout for this, so we might as well have t from day one

Reply:
Hmm, actually we currently reuse `TimeoutStartSec=` for reloading too. I think there isn't a clear separation between this operation vs reload in that regard, but both should be distinguished from start. Hence maybe introduce TimeoutReloadSec= or Refresh that covers this, and fall back to TimeoutStartSec= when unspecified?

Reply:
also sounds ok to me.

Reply:
I'll drop this then, and if one day a reload-specific timeout is added, this can be switched too